<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>p2kontakt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="list_tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_change_identity">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Identity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_userid">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fingerprint">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>fingerprint</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_chat">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>621</width>
      <height>441</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_chat">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>551</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_send">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>530</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_chatting">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Chatting with</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_server_status">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>560</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_add_contact"/>
    <addaction name="action_clear_chat"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuIdentity">
    <property name="title">
     <string>Identity</string>
    </property>
    <addaction name="action_new_identity"/>
    <addaction name="action_export_contact_file"/>
    <addaction name="action_authorise_ip"/>
    <addaction name="action_delete_identity"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_reset"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_encrypt"/>
    <addaction name="action_decrypt"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuIdentity"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_new_identity">
   <property name="text">
    <string>Create New Identity</string>
   </property>
  </action>
  <action name="action_authorise_ip">
   <property name="text">
    <string>Authorise an IP</string>
   </property>
  </action>
  <action name="action_delete_identity">
   <property name="text">
    <string>Delete Identity</string>
   </property>
  </action>
  <action name="action_add_contact">
   <property name="text">
    <string>Add a Contact</string>
   </property>
  </action>
  <action name="action_encrypt">
   <property name="text">
    <string>Encrypt</string>
   </property>
  </action>
  <action name="action_decrypt">
   <property name="text">
    <string>Decrypt</string>
   </property>
  </action>
  <action name="action_export_contact_file">
   <property name="text">
    <string>Export Contact File</string>
   </property>
   <property name="toolTip">
    <string>Export Contact File</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="text">
    <string>Reset ALL Data</string>
   </property>
  </action>
  <action name="action_clear_chat">
   <property name="text">
    <string>Clear Chat</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_chat</sender>
   <signal>returnPressed()</signal>
   <receiver>button_send</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
